db/data.py

```diff
@@ -460,3 +462,37 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def find_md5_duplicates(md5):
+    """Check if data comes from an audio file with the same md5 (Uses data->md5_encoded)
+        
+    Args: 
+        md5: The md5_encoded from the low-level data to check
+    
+    Returns:
+        The retrieved mbid if the md5 already exists
+        None if the md5 is not found
+    """
+
+    query = text(
+            """SELECT lowlevel.mbid
+                 FROM lowlevel
+                 JOIN lowlevel_json USING (id)
+                WHERE lowlevel_json.data#> '{metadata,audio_properties,md5_encoded}' ? :md5""")
+    with db.engine.begin() as connection:
+        result = connection.execute(query, {"md5": md5})
+        row = result.fetchone()
+        if row:
+            # returns the retrieved mbid
```

Comment:
This comment isn't needed. In general you should use comments to explain the reasoning behind a decision, rather than just saying in text the same thing that the statement does. We don't have any comments like this in other methods in this file.